### DISCUSSION TITLE
イベントの時間がUTCで表示されている

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#562] [要望・アイデア・不具合]
[公式イベント](https://qiita.com/official-events)のページについて、ページをロードした際、添付のように一瞬UTCで表示されます。
混乱を招きやすいため、修正をお願いします。
<img width="1464" alt="スクリーンショット 2023-11-07 11 10 07" src="https://github.com/increments/qiita-discussions/assets/38848514/6f89860a-a89d-42b5-be60-44606ff18c59">

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご報告ありがとうございます🙏
対応します！

**Comment by shimanakahibiki** [comment]
@Brutus03 
お待たせしました！
こちらロードをしても常時JSTが表示されるように修正いたしました🙇

**Reply by Brutus03** [reply]
修正確認しました。
クローズで大丈夫です。ご対応ありがとうございました。
